packages/cfd/src/Containers/mt5-migration-modal/mt5-migration-front-side-content.tsx

```diff
@@ -54,16 +81,23 @@ const MT5MigrationFrontSideContent = observer(() => {
                     )}
                 </div>
             </div>
-            <div>
+            <div className='mt5-migration-modal__message'>
                 <Text as='p' size={content_size} align='center'>
                     <Localize
-                        i18n_default_text='Click <0>Next</0> to start your transition.'
-                        components={[<strong key={0} />]}
+                        i18n_default_text='By clicking on "<0>Next</0>" you agree to move your SVG MT5 account(s) under Deriv (V) Ltd’s <1>terms and conditions</1>.'
+                        components={[
```

Comment:
we shouldnt hard code - Deriv (V) Ltd’s. It should be the DBVI_COMPANY_NAMES[getEligibleAccountToMigrate()].name